src/plugins.test.js

```diff
@@ -125,7 +125,7 @@ test('A single plugin can be loaded and work', async () => {
     const metric = _.chain(reports)
       .find(obj => obj.client_id === 'single-plugin')
       .get('custom_metrics')
-      .find({ name: 'trace-ok', s: 'neat' })
+      .find({ name: 'dummy-ok', s: 'neat' })
       .value();
     expect(_.isObject(metric)).toBe(true);
```

Comment:
@kolanos I'd like to see an additional expect added here that verifies the meta got into the report. That will ensure that the full code path regarding plugin meta was executed correctly.

Reply:
@coreylight Ok, added a more in-depth test. The meta data handling should now handle a plugin with no meta data more gracefully, although we may want to add a warning for plugins that don't have meta data. Thoughts?